sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapFeatureQueryDelegate.kt

```diff
@@ -15,8 +15,11 @@ interface MapFeatureQueryDelegate {
    * @param callback Callback called when the query completes
    */
   @Deprecated(
-    "Should be replaced with cancelable overloaded function taking RenderedQueryGeometry",
-    ReplaceWith("queryRenderedFeatures(geometry, options, callback)")
+    "Should be replaced overloaded function taking RenderedQueryGeometry and returning cancelable",
+    replaceWith = ReplaceWith(
+      "queryRenderedFeatures(RenderedQueryGeometry(pixel), options, callback).let { cancelable -> }",
```

Comment:
Such notation actually is replace from Android Studio autosuggest into working code, transforming

```
    mapboxMap.queryRenderedFeatures(
      ScreenBox(
        ScreenCoordinate(0.0, 0.0),
        ScreenCoordinate(0.0, 0.0)
      ),
      RenderedQueryOptions(null, null)
    ) {
    }
```

into 
```
    mapboxMap.queryRenderedFeatures(
      RenderedQueryGeometry(
        ScreenBox(
          ScreenCoordinate(0.0, 0.0),
          ScreenCoordinate(0.0, 0.0)
        )
      ), RenderedQueryOptions(null, null)
    ) {
    }.let { cancelable -> }
```

Reply:
Also, to me it seems it's nice to hint somehow that function actually returns cancelable handle now, so `.let {}` is added to the replaceWith. 
WDYT @mapbox/maps-android ?

Also, `val cancelable = queryRenderedFeatures(RenderedQueryGeometry(pixel), options, callback)` is not possible since assignments aren't expressions in Kotlin

Reply:
@yunikkk one thing that slightly concerns me is that we still have Java users that may not get your idea. Additionally I believe that 99% do not need actually that `Cancelable` object so it may happen that we will generate that code but they will need to clean-up that `let {}` block.

To sum up I'd go with not adding `let` block here, just go with replacing parameters.

Reply:
> `queryRenderedFeatures(RenderedQueryGeometry(pixel), options, callback)`
should be `shape` here and not `pixel`